src/helpers/forelder.ts

```diff
@@ -0,0 +1,127 @@
+import {
+  EHarSamværMedBarn,
+  EHarSkriftligSamværsavtale,
+  EHvorforIkkeOppgi,
+  EHvorMyeSammen,
+} from '../models/steg/barnasbosted';
+import { EForelder, IForelder } from '../models/forelder';
+import { isValid } from 'date-fns';
+import { ESvar, ISpørsmål, ISvar } from '../models/spørsmålogsvar';
+import { harValgtSvar } from '../utils/spørsmålogsvar';
+
+export const visBostedOgSamværSeksjon = (
+  forelder: IForelder,
+  visesBorINorgeSpørsmål: boolean
+) => {
+  const {
+    hvorforIkkeOppgi,
+    kanIkkeOppgiAnnenForelderFar,
+    ikkeOppgittAnnenForelderBegrunnelse,
+  } = forelder;
+
+  const erAnnetBegrunnelseUtfylt =
+    hvorforIkkeOppgi?.svarid === EHvorforIkkeOppgi.annet &&
+    ikkeOppgittAnnenForelderBegrunnelse?.verdi !== hvorforIkkeOppgi.verdi &&
+    ikkeOppgittAnnenForelderBegrunnelse?.verdi !== '';
+
+  const kanIkkeOppgiDenAndreForelderen =
+    kanIkkeOppgiAnnenForelderFar && hvorforIkkeOppgi
+      ? kanIkkeOppgiAnnenForelderFar?.verdi &&
+        (hvorforIkkeOppgi?.svarid === EHvorforIkkeOppgi.donorbarn ||
+          erAnnetBegrunnelseUtfylt)
+      : false;
+
```

Comment:
Hvis `kanIkkeOppgiDenAndreForelderen` er huket av, og begrunnelse (multisvarspørsmål) for hvorfor man ikke kan oppgi den andre forelderen er valgt.. 

**Dersom svaret valgt er "donorbarn"** 

![Skjermbilde 2020-06-02 kl  09 26 05](https://user-images.githubusercontent.com/8249453/83492145-1e240600-a4b3-11ea-966e-532a91e5915b.png)

**ELLER "annet" OG "annet"-fritekstfeltet er utfylt...** 

![Skjermbilde 2020-06-02 kl  09 25 50](https://user-images.githubusercontent.com/8249453/83492140-1cf2d900-a4b3-11ea-9c61-645a42c3fbde.png)


Hvis neste spørsmål :) Som handler om delt bosted: 

![Skjermbilde 2020-06-02 kl  09 28 01](https://user-images.githubusercontent.com/8249453/83492309-63483800-a4b3-11ea-9124-796337c95d38.png)